ci/prod/deploy-pipeline.yaml

```diff
@@ -435,3 +435,95 @@ spec:
+              CLOUDHSM_IP: ((cluster.cloudHsmIp))
+            run:
+              path: /bin/bash
+              args:
+                - -euc
+                - |
+                  echo "preparing keyring..."
+                  echo "${CLUSTER_PRIVATE_KEY}" > key
+                  gpg --import key
+                  gpg --export-secret-keys > ~/.gnupg/pubring.gpg
+                  KEY_ID="$(gpg --list-secret-keys --with-colons  | awk -F: '/uid:/ {print $10}' | head -n1)"
+                  echo "verifying package"
+                  helm verify ./release/*.tgz
+                  echo "rendering chart with release name '${RELEASE_NAME}' and namespace '${RELEASE_NAMESPACE}'..."
+                  helm template \
+                    --name "${RELEASE_NAME}" \
+                    --namespace "${RELEASE_NAMESPACE}" \
+                    --set "global.cluster.name=${CLUSTER_NAME}" \
+                    --set "global.cluster.domain=${CLUSTER_DOMAIN}" \
+                    --set "global.cloudHsm.ip=${CLOUDHSM_IP}" \
+                    --set "connector.entityID=${CONNECTOR_ENTITY_ID}" \
+                    --set "connector.metadata.fqdn=${CONNECTOR_METADATA_FQDN}" \
+                    --set "connector.metadata.path=${CONNECTOR_METADATA_PATH}" \
+                    --set "connector.metadataSigningTruststoreBase64=${CONNECTOR_METADATA_SIGNING_TRUSTSTORE_BASE64}" \
+                    --set "translator.connectorNodeNationalityCode=${CONNECTOR_NODE_NATIONALITY_CODE}" \
+                    --output-dir "./manifests/" \
+                    ./release/*.tgz
```

Comment:
I guess there's no easy way to script these?

Reply:
No :(
But we will refactor PN config this quarter